frontend/packages/console-shared/src/components/catalog/CatalogController.tsx

```diff
@@ -106,7 +111,10 @@ const CatalogController: React.FC<CatalogControllerProps> = ({
     });
     const crumbs = [
       {
-        name: t('console-shared~Developer Catalog'),
+        name:
+          activePerspective === 'dev'
+            ? t('console-shared~Developer Catalog')
+            : t('console-shared~Software Catalog'),
```

Comment:
I'm still uncertain about having different names for the same page in the two perspectives. I'd lean toward using the same label everywhere unless we have a really good reason.

cc @kevinhatchoua

Reply:
I will keep the Developer catalog for now in both perspectives. Once it is finalize then we will update it in the followup PR.